public/500.html

```diff
@@ -97,7 +97,7 @@ <h3 class="landmark heading">Site Navigation</h3>
     <div id="main" class="system errors error-500 region">
       <h2 class="heading">Error 500</h2>
       <h3 class="heading">We're sorry, but something went wrong.</h3>
-      <p>We've been notified about this issue and we'll take a look at it shortly.</p>
+      <p>If you are receiving this error repeatedly, please <%= link_to ts('contact Support'), new_feedback_report_path %>. In the form, please include a link to the page you're trying to reach and how you're trying to reach this page.</p>
```

Comment:
Since this is a plain old static page, the link here will need to be just regular HTML (preferably just "/support" rather than the full URL), not Ruby. 

Sorry, my fault for not explicitly saying that!